packages/flutter_test/lib/src/window.dart

```diff
@@ -683,6 +750,11 @@ class TestPlatformDispatcher implements ui.PlatformDispatcher {
     _initialLifecycleStateTestValue = state;
   }
 
+  /// Resets [initialLifecycleState] to the default value for the platform.
+  void resetInitialLifecycleState() {
```

Comment:
call this from clearAllTestValues? (and maybe double check that that method calls all the other ones?)

Reply:
That was _one_ of the nice things about using the configuration. The reset method was super straight forward and resistant to forgetting new reset methods...

Still worth it to remove, but still.

Reply:
I was wondering if one could write a test to enforce that clearAllTestValues clears all values (including ones added in the future). Couldn't come up with anything good there, though...

Reply:
I've manually made sure this is called for the moment.

I _think_ I could write a test to validate clearing all of the values, but it'd take me a bit, and I'm not _sure_ if I could do it without `mirrors`. I may take a poke at it at some point, but not right now.